.circleci/config.yml

```diff
@@ -22,6 +22,7 @@ workflows:
             branches:
               only:
                 - master
+                - add-pushed-at-field
```

Comment:
Revert?

Reply:
Yup, its in a separate commit. Just for me to have a temporary version test if it helps